python/taichi/profiler/kernel_profiler.py

```diff
@@ -199,6 +199,7 @@ def _clear_frontend(self):
     def _update_records(self):
         """Acquires kernel records from a backend."""
         impl.get_runtime().prog.sync_kernel_profiler()
+        impl.get_runtime().prog.update_kernel_profiler()
```

Comment:
Somehow I feel this should be `prog.kernel_profiler().sync()` lol but that's a totally different issue.

Reply:
`sync` used to be sync+update... (reuse the old name in code), now `sync` is sole `sync`, and `update` becomes `update` ;// a bit hard to explain